### PR TITLE
Feature/T-4: Create Add new delivery request page

### PR DESCRIPTION
**Pull request by crisan-catalin**

### CODE REVIEW

File: src/main/java/com/example/fooddelivery/model/Address.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
     @NotNull
     @Column
-    private Integer number;
+    private String number;
```
Comment (LunguNicolae) [inline src/main/java/com/example/fooddelivery/model/Address.java:31]: why you changed the type of number?

File: src/main/java/com/example/fooddelivery/forms/AddressForm.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.example.fooddelivery.forms;
+
+public class AddressForm {
+    private String city;
+    private String street;
+    private String number;
```
Comment (LunguNicolae) [inline src/main/java/com/example/fooddelivery/forms/AddressForm.java:6]: why don't you use integer or long?

File: src/main/java/com/example/fooddelivery/repository/ProductRepository.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.example.fooddelivery.repository;
+
+import com.example.fooddelivery.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+    List<Product> findAllByRestaurant_Id(Long id);
```
Comment (LunguNicolae) [inline src/main/java/com/example/fooddelivery/repository/ProductRepository.java]: why you used '_' in method signature? can't we use only camel case?

File: src/main/java/com/example/fooddelivery/model/Address.java (hunk 1 above)
Comment (crisan-catalin) [reply src/main/java/com/example/fooddelivery/model/Address.java:31]: House numbers for example. We can have nr. 4a

File: src/main/java/com/example/fooddelivery/repository/ProductRepository.java (hunk 3 above)
Comment (crisan-catalin) [reply src/main/java/com/example/fooddelivery/repository/ProductRepository.java]: Done.

File: src/main/java/com/example/fooddelivery/forms/AddressForm.java (hunk 2 above)
Comment (crisan-catalin) [reply src/main/java/com/example/fooddelivery/forms/AddressForm.java:6]: For House numbers for example. We can have nr. 4a

**Comment by LunguNicolae** [comment]
As we discussed this PR could have conflicts, and it does. @crisan-catalin could you checkout it, see what is the problem and then merge it? I would do the merge by myself but don't know exactly what changes to keep

**Pull request merged**